### ISSUE TITLE
v183+ GMS incorrect outbound opcode

### ISSUE THREAD

**Issue by fsdjfhdsfj** [diamondo25/MapleShark#20]
This issue has been happening since v183.1.

Unable to see the correct outbound opcode. 
The packet structure seems fine.
The opcode changes each time upon channel change. 
It seems to affect only the outbound opcodes that are used for the channels. 
The login outbound opcodes are proper and don't change.

**Comment by diamondo25** [comment]
Yes, Nexon has implemented an opcode crypto. I am currently unable to help with this issue, as it would require some major changes to the codebase, possibly adding realtime packet handling (would help with logging things like drops in realtime, too!).

**Comment by BobNigers** [comment]
can you provide code for opcode crypto like from ida?

**Comment by diamondo25** [comment]
Nope, because its obfuscated.

**Comment by gigibader** [comment]
Now that a v188 Unvirtualized Client has been released will this issue be fixed?
